R/param-estimates.R

```diff
@@ -53,11 +77,44 @@ param_estimates.bbi_nonmem_summary <- function(.summary) {
     ) %>%
     tibble::as_tibble()
 
-  param_df[["diag"]] <- map_lgl(param_df[["names"]], is_diag)
+  param_df[["fixed"]] <- map_lgl(param_df[["fixed"]], ~ .x == 1)
```

Comment:
Because `==` is vectorized, this could be more simply expressed as `param_df[["fixed"]] == 1`.